README.md

```diff
@@ -36,22 +36,31 @@ jobs:
         with:
           token: ${{ github.token }}
           branch: main
+          major-list: break
+          minor-list: feat, feature
+          patch-list: fix, bugfix, perf, refactor, test, tests
 
       - name: Create Release
         uses: ncipollo/release-action@v1
         with:
           allowUpdates: true
           draft: false
           name: ${{ steps.semver.outputs.next }}
+          tag: ${{ steps.semver.outputs.next }}
```

Comment:
Again, this line should be removed as it could potentially cause issues for some users that use a pre-existing tag that doesn't match this value. The default of using the ref is better suited for the example code.